examples/temperature-sensor/temperature-publisher/modules/device/Module.md

```diff
@@ -0,0 +1,6 @@
+Prints "Hello, World!" with a main function.
```

Comment:
Do we need this file ? If not shall we remove this/

Reply:
Fixed